### PR TITLE
Fix and add Russian and Ukrainian words

### PR DESCRIPTION
**Pull request by dejurin**
Changes proposed in this pull request:

* Add new words
* Corrected the endings for the words

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks for the PR!

The last two groups need `%d` adding to the string in the place where the number should go. Please can you update them?

**Comment by dejurin** [comment]
> Thanks for the PR!
> 
> The last two groups need `%d` adding to the string in the place where the number should go. Please can you update them?
please, explain

File: src/humanize/locale/ru_RU/LC_MESSAGES/humanize.po [hunk 1]
```diff
@@ -248,17 +248,17 @@ msgstr "девять"
 #, python-format
 msgid "%d microsecond"
 msgid_plural "%d microseconds"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "микросекунда"
+msgstr[1] "микросекунды"
+msgstr[2] "микросекунд"
```
Comment (hugovk) [inline src/humanize/locale/ru_RU/LC_MESSAGES/humanize.po:253]: The `%d` will be replaced by a number. So in English:

"%d microsecond" -> "1 microsecond"
"%d microseconds" -> "2 microseconds"

Is this correct?

```suggestion
msgstr[0] "%d микросекунда"
msgstr[1] "%d микросекунды"
msgstr[2] "%d микросекунд"
```

File: src/humanize/locale/ru_RU/LC_MESSAGES/humanize.po [hunk 2]
```diff
@@ -248,17 +248,17 @@ msgstr "девять"
...
 #: src/humanize/time.py:142
 #, python-format
 msgid "%d millisecond"
 msgid_plural "%d milliseconds"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "миллисекунда"
+msgstr[1] "миллисекунды"
+msgstr[2] "миллисекунд"
```
Comment (hugovk) [inline src/humanize/locale/ru_RU/LC_MESSAGES/humanize.po:261]: And like this?

```suggestion
msgstr[0] "%d миллисекунда"
msgstr[1] "%d миллисекунды"
msgstr[2] "%d миллисекунд"
```

**Review by hugovk — COMMENTED** [review]
Hi, like this:

**Comment by dejurin** [comment]
oh... correct

**Comment by hugovk** [comment]
The PyPy3/macOS failure is unrelated, I'll fix that separately. Merging, thank you!

**Pull request merged**